README.md

```diff
@@ -244,7 +171,11 @@ bin/console upgradability:php:upgrade
 export INTEGRATOR_ENABLED=true
 ```
 
-​
+## Evaluator
+
+[Evaluator documentation](https://github.com/spryker-sdk/upgrader/blob/master/docs/evaluator.md)
```

Comment:
For links on the docs inside the same repo use the relative paths: 

[Evaluator documentation](docs/evaluator.md).